### ISSUE TITLE
Fix locl for bulgarian and serbian/macedonian Cyrillic

### ISSUE THREAD

**Issue by davelab6** [TypeNetwork/Roboto-Extremo#362]
The localization features for bulgarian and serbian/macedonian Cyrillic are currently commented out:

https://github.com/googlefonts/roboto-flex/blob/7dd9b0b48ef0d2f6001e7f1478184d393cc3e138/sources/1A-drawings/features/RobotoFlex.fea#L10-L11

Please explain why these essential language support features weren't included in the last release, and the plan and schedule to bring them back :)

**Comment by sannorozco** [comment]
Yes, there was this issue where all the features worked, except these localizations. The VF compiled and everything, no error or warning, they just didn't’t work. I’m debugging this for the next release.
